platforms/.eslintignore

```diff
@@ -7,3 +7,4 @@
 /src/didkit-browser/*
 /__mocks__/**/*
 /__tests__/**/*
+/src/Snapshot/__tests__/**/*
```

Comment:
I would prefer not to add too many if our own source files to the linter ignore list.